packages/frontend-core/src/components/grid/cells/AttachmentSingleCell.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+  import AttachmentCell from "./AttachmentCell.svelte"
+
+  export let value
+  export let onChange
+
+  $: arrayValue = (!Array.isArray(value) && value ? [value] : value) || []
+
+  $: onFileChange = value => {
+    value = value[0] || null
+    onChange(value)
+  }
+</script>
+
+<AttachmentCell
+  {...$$restProps}
+  maximum={1}
```

Comment:
We'll need to `bind:api` here to ensure the bind is properly passed through (which makes things like keyboard event or programatic cell focus/blur work), so we'll need to add an explicit `export let api` to facilitate that.